appveyor.yml

```diff
@@ -10,6 +10,8 @@ install:
   - rmdir c:\go /s /q
   - appveyor DownloadFile https://storage.googleapis.com/golang/go%GOVERSION%.windows-amd64.zip
   - 7z x go%GOVERSION%.windows-amd64.zip -y -oC:\ > NUL
+  - cd %APPVEYOR_BUILD_FOLDER%
+  - git submodule update --init --recursive
```

Comment:
for my understanding, why are we adding this here?

Reply:
Yeah, was wondering why we were running these tests in Windows. later realized we run all of the plugin tests in both platforms too.